src/MongoDB.Driver.Core/MongoConnectionException.cs

```diff
@@ -78,6 +80,24 @@ public ConnectionId ConnectionId
             get { return _connectionId; }
         }
 
+        /// <summary>
+        /// Whether or not this exception contains a timeout exception.
+        /// </summary>
+        public bool ContainsTimeOutException
+        {
+            get
+            {
+                for (var inner = InnerException; inner != null; inner = inner.InnerException)
```

Comment:
`inner` is an adjective, not a noun.

Rename to `exception`, or if you prefer a longer name I would also accept `innerException`.

Reply:
Done!